contrib/coreos/cloud-config

```diff
@@ -0,0 +1,266 @@
+#cloud-config
```

Comment:
Yes, which is why this file should be moved elsewhere or else `make discovery-url` will overwrite it. This file is in gitignore: https://github.com/deis/deis/blob/master/.gitignore#L68-L69